pennylane/beta/plugins/default_tensor.py

```diff
@@ -355,41 +366,73 @@ class DefaultTensor(Device):
     C_DTYPE = np.complex128
     R_DTYPE = np.float64
 
-    def __init__(self, wires, shots=1000, analytic=True):
+    zero_state = np.array([1.0, 0.0], dtype=C_DTYPE)
+
+    def __init__(self, wires, shots=1000, analytic=True, representation="exact"):
```

Comment:
Nice!

Reply:
Don't forget to update the docstring with details on `representation`

Reply:
Gonna leave this for the future PR where I add the MPS representation